requestStorageAccessForOrigin.bs

```diff
@@ -0,0 +1,203 @@
+<pre class="metadata">
+Title: requestStorageAccessForOrigin API
+Shortname: storage-access-for-origin
+Abstract: The requestStorageAccessForOrigin API allows top-level sites to request access to cross-site cookies on behalf of embedded origins.
+Status Text: This specification is intended to be merged into the HTML Living Standard. It is neither a WHATWG Living Standard nor is it on the standards track at W3C.
+Text Macro: LICENSE <a href=https://creativecommons.org/licenses/by/4.0/>Creative Commons Attribution 4.0 International License</a>
+Group: None
+ED: https://github.com/mreichhoff/requestStorageAccessForOrigin
+Status: w3c/UD
+Editor: Matt Reichhoff, w3cid 138889, Google https://google.com, mreichhoff@google.com
+Editor: Johann Hofmann, w3cid 120436, Google https://google.com, johannhof@google.com
+Level: None
+Markup Shorthands: markdown yes, css no
+Complain About: accidental-2119 true
+</pre>
+
+<pre class=link-defaults>
+spec:html; type:dfn; for:site; text:same site
+spec:webidl; type:dfn; text:resolve
+</pre>
+
+<pre class="anchors">
+urlPrefix: https://url.spec.whatwg.org/; spec:url;
+    type:dfn
+        text:origin
+urlPrefix: https://tc39.github.io/ecma262/; spec: ECMASCRIPT
+    text: agent cluster; url: #sec-agent-clusters; type: dfn
+urlPrefix: https://infra.spec.whatwg.org/; spec: INFRA
+    text: implementation-defined; url: #implementation-defined; type: dfn
+urlPrefix: https://privacycg.github.io/storage-access/#; spec: storage-access
+    type: dfn
+        text: determine the storage access policy; url: determine-the-storage-access-policy
+        text: determine if a site has storage access; url: determine-if-a-site-has-storage-access
+urlPrefix: https://fetch.spec.whatwg.org/#; spec: fetch
+    type: dfn
+        text: http network or cache fetch; url: http-network-or-cache-fetch
+        text: request; url: concept-request
+</pre>
+
+<section class="non-normative">
+<h2 id="intro">Introduction</h2>
+
+<em>This section is non-normative.</em>
+
+Many User Agents prevent content from accessing non-[=same site=] data stored in cookies. 
+This can break embedded content which relies on having access to non-[=same site=] cookies.
+
+The requestStorageAccessForOrigin API enables developers to request access to non-[=same site=] cookies for embedded resources such as iframes, scripts, or images.
+It accomplishes this by specifying {{Document/requestStorageAccessForOrigin(origin)}}, which allows top-level browsing contexts to request access
```

Comment:
I'd lean towards using "navigable" instead of "browsing context" these days. Specifically, I'd use "traversable navigable" here instead of top-level browsing. Your options are "traversable navigable" and "top-level traversable" here, and I would use the former which is a superset of the latter. And I'd also link to it: https://html.spec.whatwg.org/multipage/document-sequences.html#traversable-navigable.

The reason I recommend using traversable navigable over top-level traversable is because you probably want to treat i.e., fenced frames and portals as "top-level" in this context. Those frames are rooted at a traversable navigable with a non-null parent.

Reply:
switched to that in the latest version